pkg/k8s/virtualservices/client.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package virtualservices
+
+import (
+	"github.com/okteto/okteto/pkg/okteto"
+	istioclientset "istio.io/client-go/pkg/clientset/versioned"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// GetIstioClient returns a client for istio
+func GetIstioClient() (*istioclientset.Clientset, error) {
+	octx := okteto.Context()
+	clientConfig := clientcmd.NewDefaultClientConfig(*octx.Cfg, nil)
```

Comment:
NIT: You can get the `restConfig` directly using the `k8sProvider`. This will make easier to test later all these functions

```suggestion
	client, config, err := okteto.NewK8sClientProvider().Provide(okteto.Context().Cfg)
```